templates/devenv.yaml.tpl

```diff
@@ -0,0 +1,17 @@
+dependencies:
+{{- if gt (len (stencil.GetModuleHook "devenv.dependencies.optional")) 0 }}
+	{{- range (stencil.GetModuleHook "devenv.dependencies.optional") }}
+	optional:
+		- "{{ . }}"
+	{{- end }}
+{{- else }}
+	optional: []
+{{- end }}
+{{- if gt (len (stencil.GetModuleHook "devenv.dependencies.required")) 0 }}
```

Comment:
Unncessary if, range will noop if no values

Reply:
i need to put an empty array in the yaml file if it has no values